Rakefile

```diff
@@ -19,104 +20,103 @@ begin
 rescue LoadError => _
 end
 
-namespace :device_agent do
-
-  desc "Install DeviceAgent binaries"
-  task :install => [:clean, :build, :expand]
+def colorize(string, color)
+  "\033[#{color}m#{string}\033[0m"
+end
 
-  def colorize(string, color)
-    "\033[#{color}m#{string}\033[0m"
-  end
+def cyan(string)
+  colorize(string, 36)
+end
 
-  def cyan(string)
-    colorize(string, 36)
-  end
+def green(string)
+  colorize(string, 32)
+end
 
-  def green(string)
-    colorize(string, 32)
-  end
+def magenta(string)
+  colorize(string, 35)
+end
 
-  def magenta(string)
-    colorize(string, 35)
-  end
+def log_unix_cmd(msg)
+  puts cyan("EXEC: #{msg}") if msg
+end
 
-  def log_unix_cmd(msg)
-    puts cyan("EXEC: #{msg}") if msg
-  end
+def log_info(msg)
+  puts green("INFO: #{msg}") if msg
+end
 
-  def log_info(msg)
-    puts green("INFO: #{msg}") if msg
-  end
+def banner(msg)
+  puts ""
+  puts magenta("######## #{msg} ########")
+  puts ""
+end
 
-  def banner(msg)
-    puts ""
-    puts magenta("######## #{msg} ########")
-    puts ""
+def device_agent_dir
+  @device_agent_dir ||= begin
+    dir = File.join(".", "lib", "run_loop", "device_agent")
+    File.expand_path(dir)
   end
+end
 
-  def device_agent_dir
-    @device_agent_dir ||= begin
-      dir = File.join(".", "lib", "run_loop", "device_agent")
-      File.expand_path(dir)
-    end
-  end
+def frameworks_dir
+  @frameworks_dir ||= File.join(device_agent_dir, "Frameworks")
+end
 
-  def frameworks_dir
-    @frameworks_dir ||= File.join(device_agent_dir, "Frameworks")
-  end
+def frameworks_zip
+  @frameworks_zip ||= "#{frameworks_dir}.zip"
+end
 
-  def frameworks_zip
-    @frameworks_zip ||= "#{frameworks_dir}.zip"
-  end
+def app_dir
+  @app_dir ||= File.join(device_agent_dir, "app", "DeviceAgent-Runner.app")
+end
 
-  def app_dir
-    @app_dir ||= File.join(device_agent_dir, "app", "DeviceAgent-Runner.app")
-  end
+def app_zip
+  @app_zip ||= "#{app_dir}.zip"
+end
 
-  def app_zip
-    @app_zip ||= "#{app_dir}.zip"
-  end
+def ipa_dir
+  @ipa_dir ||= File.join(device_agent_dir, "ipa", "DeviceAgent-Runner.app")
+end
 
-  def ipa_dir
-    @ipa_dir ||= File.join(device_agent_dir, "ipa", "DeviceAgent-Runner.app")
-  end
+def ipa_zip
+  @ipa_zip ||= "#{ipa_dir}.zip"
+end
 
-  def ipa_zip
-    @ipa_zip ||= "#{ipa_dir}.zip"
-  end
+def bin
+  @bin ||= File.join(device_agent_dir, "bin", "iOSDeviceManager")
+end
 
-  def bin
-    @bin ||= File.join(device_agent_dir, "bin", "iOSDeviceManager")
-  end
+def license
+  @license ||= File.join(device_agent_dir, "bin", "iOSDeviceManager.LICENSE")
```

Comment:
This line sounds like "Take `iOSDeviceManager.LICENSE` from DeviceAgent folder". Could you please explain this behavior?

Reply:
Previously `:clean` task has not delete `iOSDeviceManager.LICENSE` file. I've added this def to delete it in `iOSDeviceManager:clean` task